### DISCUSSION TITLE
New Release UI

### DISCUSSION THREAD

**Discussion by proneon267** [community/community#6084] [Other Feature Feedback, Questions, & Ideas] [answered]
The new releases UI takes more time to get to the actual release files, as first the release name needs to be clicked on and then it shows the release files. It should just show the release files along with their names in a single page, like in old releases UI

**Comment by MylesBorins** [comment, marked as answer]
We've updated the UI to now show all artifacts in the list view and when you click "read more" it loads inline instead of forwarding you to a new page.

Thanks for the feedback!

**Reply by proneon267** [reply]
Looks and works great now!
